runtime/ztests/expr/queryexpr-multiple-err.yaml

```diff
@@ -0,0 +1,6 @@
+spq: values (values 1,2,3)
+
+vector: true
+
+output: |
+  error({message:"encountered 3 values in expression subquery (consider collect())",on:1})
```

Comment:
This message doesn't make sense. It says there are 3 values but the `on` field only contains a single value.